av_transducer/calculators/asr_calculator.cc

```diff
@@ -0,0 +1,70 @@
+#include "mediapipe/framework/api2/node.h"
+#include "mediapipe/framework/api2/packet.h"
+#include "ml/asr/model.h"
+#include <vector>
+
+namespace aikit {
+
+class ASRCalculator : public mediapipe::api2::Node {
+public:
+    static constexpr mediapipe::api2::SideInput<std::string> kInASRModelPath{
+        "ASR_MODEL_PATH"};
+    static constexpr mediapipe::api2::SideInput<std::string> kInSPKModelPath{
+        "SPK_MODEL_PATH"};
+    static constexpr mediapipe::api2::SideInput<int> kInBufferDurationSec{
```

Comment:
You can make it optional by marking like this:
`static constexpr mediapipe::api2::SideInput<int>::optional kInBufferDurationSec`